### ISSUE TITLE
Nodes initialised with isHidden: true remains invisible after toggling to false

### ISSUE THREAD

**Issue by JamesyJi** [wbkd/react-flow#506]
Here is a very basic example code with 2 nodes. One is hidden at the start. Clicking on the button should toggle visbility.

https://codesandbox.io/s/peaceful-robinson-vr4gt?file=/src/App.js

Toggling it hides Node 2 but does not show Node 1 at the top. The thing is, if you move your mouse to near the top of the screen (to where Node 1 is, set isHidden to false to get an idea of its position), you are able to click on Node 1, implying that it exists. However, it is just invisible.

This only seems to happen if isHidden was true when the entire ReactFlow was initialised. In my personal project, I had no problems toggling isHidden if the node's isHidden was initially set to false.

**Comment by moklick** [comment]
This seems like a bug. I will look into it.

**Comment by ynsmrkrkmz** [comment]
I have the same issue now. I use v8.8.0. I save isidden data on localstorage and when I refresh the page it works. I have approximately 20 nodes I am trying to filter them when I change to filter state ON four of the nodes should be visible but only one of them becomes visible. I checked the json creating nodes all values are true actually. I checked nodes bu using F12 button their opacity values are 0 but they should be visible actually.
